themes/_core/js/product.js

```diff
@@ -49,14 +49,15 @@ var formChanged = false;
  */
 function getProductUpdateUrl() {
   let dfd = $.Deferred();
-  const $productActions = $('.product-actions');
-  const $quantityWantedInput = $('#quantity_wanted');
-
-  if (prestashop !== null
-    && prestashop.urls !== null
-    && prestashop.urls.pages !== null
-    && prestashop.urls.pages.product !== ''
-    && prestashop.urls.pages.product !== null
+  const $productActions = $(prestashop.selectors.product.actions);
+  const $quantityWantedInput = $(prestashop.selectors.quantityWanted);
+
+  if (
+    prestashop !== null &&
+    prestashop.urls !== null &&
+    prestashop.urls.pages !== null &&
+    prestashop.urls.pages.product !== '' &&
+    prestashop.urls.pages.product !== null
```

Comment:
&& at the start ;)

Reply:
There are no linters on this part until https://github.com/PrestaShop/PrestaShop/pull/20080, shouldn't be blocking